IpmiFeaturePkg/Include/Library/IpmiSelLib.h

```diff
@@ -113,6 +85,27 @@ SelAddOemEntry (
   IN UINT8       Data[6]
   );
 
+/**
+  Adds an OEM timestamped event to the SEL.
+
+  @param[in,out]  RecordId         If provided, receives the record ID of the entry.
+  @param[in]      RecordType       The record type code. But be between 0xC0-0xDF.
+  @param[in]      ManufacturerId   The manufacturer ID for the event. Must be 3 bytes.
+  @param[in]      Data             Array of OEM defined event data.
+
+  @retval   EFI_SUCCESS             Event was successfully added to the SEL.
+  @retval   EFI_INVALID_PARAMETER   Invalid RecordType was given.
+  @retval   Other                   And error was returned by IpmiAddSelEntry.

```

Comment:
I suggest dropping implementation details (like calling `IpmiAddSelEntry`) from the interface documentation.